tripy/examples/segment-anything-model-v2/sam2/modeling/memory_attention.py

```diff
@@ -124,7 +124,7 @@ def __init__(
     ):
         super().__init__()
         self.d_model = d_model
-        self.layers = [layer for i in range(num_layers)]
+        self.layers = get_clones(layer, num_layers)
```

Comment:
@parthchadha @pranavm-nvidia I met a problem here, `get_clones` uses `deepcopy` to copy the `layer` module multiple times, but `runtime.MemrefValue` is not pickle-able, thus the deepcopy failed. This is because we always create memref in `StorageOp` now. I had to duplicate the `layer` target definition in the config file to work around it, is there a better approach?

Reply:
I am not really sure why cloning is even required here.

Reply:
I think it's because the weights are different. `[layer for i in range(num_layers)]` does not create distinct modules.

Reply:
Clones also has this problem that it forces Tripy to create exact same names for all tensors whereas if you were initializing all layers here then you would get unique intermediate layer names.

Reply:
@yizhuoz004 can we implement `__deepcopy__` for `StorageOp` instead of modifying the yaml files and layer structure?